.eslintrc.json

```diff
@@ -21,11 +19,12 @@
     "plugin:@typescript-eslint/recommended",
     "plugin:react-hooks/recommended",
     "prettier/@typescript-eslint",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
+    "plugin:jsx-a11y/recommended"
```

Comment:
What is this change? (is this related somehow to this PR?

Reply:
Yes.
The emoji rule did not like that I am setting the role + aria-label on something other than a span (it's div Styled component).
I'm disabling that rule in the code block where this happens, but vscode doesn't know about the rule without this change, and thinks the code is broken otherwise.